staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -99,6 +99,10 @@ type Event struct {
 	// Source IPs, from where the request originated and intermediate proxies.
 	// +optional
 	SourceIPs []string
+	// UserAgent is an optional field that specifies the caller of this request.
```

Comment:
nit: drop "is an optional field" - it's redundant.

How about: `UserAgent records the user agent string reported by the client.`

Reply:
SGTM